src/github.com/getlantern/flashlight/proxied/proxied.go

```diff
@@ -1,4 +1,8 @@
-package util
+// Package proxied provides  http.Client implementations that use various
+// combinations of chained and direct domain-fronted proxies.
+//
+// Remember to call SetProxyAddr before obtaining an http.Client.
+package proxied
```

Comment:
All the http-related stuff from util is now in the proxied package.  Fundamentally, it works the same, but it exposes a common and consistently named set of functions that all return an *http.Client in lieu of our custom HTTPFetcher interface (which is no longer necessary).